.github/workflows/build-test.yml

```diff
@@ -70,8 +70,8 @@ jobs:
       - name: Restore build artifacts
         uses: ./.github/actions/build-artifacts/restore
 
-      - name: Test
-        run: yarn test:ci:ui
+      - name: Start UI and Test
+        run: yarn start-test 'dev:ui' http://localhost:3000 'yarn test:ci:ui'
```

Comment:
Ideally we'd run the prod version here with `yarn start` but not a blocker

Reply:
`start-test` is an alias for the concurrent yarn run command (`yarn start-server-and-test`). 
Updated this for clarity.